bin/start

```diff
@@ -13,6 +13,12 @@ if [[ ! $SERVER_PORT ]]; then
   export SERVER_PORT=5000
 fi
 
+if [ -z "$DECENTRALAND_DATA" ]; then
```

Comment:
Whatever makes you happy :) I've updated it.
I would recommend to use --arguments instead of environment variables, but doing that in bash it's not so fun. But I guess this is not something urgent for you now, lots of other interesting things to do first.

Thanks for reviewing!